compositor_render/src/registry.rs

```diff
@@ -35,7 +35,7 @@ impl RegistryType {
     }
 }
 
-pub(crate) struct RendererRegistry<T: Clone> {
+pub struct RendererRegistry<T: Clone> {
```

Comment:
Just like `Renderers`, it shouldn't be public. I made it `pub(crate)` on my local branch and there were no compile errors so I'm assuming it's not needed